src/main/java/com/vtigercrm/objectrepositoryLib/Home_Page.java

```diff
@@ -1,6 +1,29 @@
 package com.vtigercrm.objectrepositoryLib;
 
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
 public class Home_Page 
 {
+	@FindBy(linkText = "Organizations")
+	private WebElement Orglink;
+	
+	@FindBy(linkText = "Contacts")
+	private WebElement Contactlink;
+	
+	public Home_Page(WebDriver driver) 
+	{
+		PageFactory.initElements(driver, this);
+	}
+
+	public WebElement getOrglink() {
+		return Orglink;
+	}
 
+	public WebElement getContactlink() {
+		return Contactlink;
+	}
+	

```

Comment:
Create business specific method
```suggestion

	public WebElement getOrglink() {
		return Orglink;
	}

	public WebElement getContactlink() {
		return Contactlink;
	}
	

```

Reply:
k will modify

Reply:
have modified code please approve and merge